cloudbuild.yaml

```diff
@@ -33,35 +33,26 @@ steps:
     path: /root/.ssh
 
 # Run the build
-- name: l.gcr.io/google/bazel:3.5.0
+- name: gcr.io/bazel-public/bazel:5.1.0
   entrypoint: 'bash'
   args:
   - '-c'
   - |
     set -e
-    apt-get update
-    apt-get install apt-transport-https ca-certificates -y
-    update-ca-certificates
-    # This is kind of a hack to upgrade bazel to 4.2.1. When a new Bazel Docker container is released https://github.com/bazelbuild/continuous-integration/issues/1060
-    # Remove this, and just bump the docker container.
-    wget https://github.com/bazelbuild/bazel/releases/download/4.2.1/bazel_nojdk-4.2.1-linux-x86_64
-    chmod +x bazel_nojdk-4.2.1-linux-x86_64
-    mv bazel_nojdk-4.2.1-linux-x86_64 /usr/local/lib/bazel/bin/bazel 
-    # End Bazel upgrade hack.
-    python3 tools/generate_uuid.py > build_invocation_id.txt
-    python3 tools/generate_uuid.py > test_invocation_id.txt
+    export BUILD_INVOCATION_ID=$(python3 tools/generate_uuid.py)
+    export TEST_INVOCATION_ID=$(python3 tools/generate_uuid.py)
     echo STATUS: Reporting logs with invocation id $(cat invocation_id.txt) to GitHub.
```

Comment:
`invocation_id.txt` probably does not exist. Not sure why it isn't failing on that